src/components/LoanCards/BorrowerInfo/BorrowerInfoBody.vue

```diff
@@ -69,13 +70,15 @@ export default {
 			const maxLength = 1000;
 			const lowerCaseUse = this.use.toString().charAt(0).toLowerCase() + this.use.toString().slice(1);
 			const convertedUse = (this.use.substring(0, this.name.length) === this.name) ? this.use : lowerCaseUse;
-
+			/* eslint-disable */
+			// TODO: Fix eslint rule to allow else if
```

Comment:
I think the rule is just complaining because the else if is not needed when the first if returns something

Reply:
resolved